rasa/nlu/training_data/training_data.py

```diff
@@ -232,27 +236,21 @@ def sort_regex_features(self) -> None:
             self.regex_features, key=lambda e: "{}+{}".format(e["name"], e["pattern"])
         )
 
-    def fill_response_phrases(self) -> None:
+    def _fill_response_phrases(self) -> None:
         """Set response phrase for all examples by looking up NLG stories"""
         for example in self.training_examples:
-            response_key = example.get(RESPONSE_KEY_ATTRIBUTE)
             # if response_key is None, that means the corresponding intent is not a
             # retrieval intent and hence no response text needs to be fetched.
             # If response_key is set, fetch the corresponding response text
-            if response_key:
-                # look for corresponding bot utterance
-                story_lookup_intent = example.get_combined_intent_response_key()
-                assistant_utterances = self.nlg_stories.get(story_lookup_intent, [])
-                if assistant_utterances:
-                    # selecting only first assistant utterance for now
-                    example.set(RESPONSE, assistant_utterances[0])
-                else:
-                    raise ValueError(
-                        "No response phrases found for {}. Check training data "
-                        "files for a possible wrong intent name in NLU/NLG file".format(
-                            story_lookup_intent
-                        )
-                    )
+            if example.get(RESPONSE_KEY_ATTRIBUTE) is None:
+                continue
+
+            # look for corresponding bot utterance
+            story_lookup_intent = example.get_combined_intent_response_key()
+            assistant_utterances = self.responses.get(story_lookup_intent, [])
+            if assistant_utterances:
+                # selecting only first assistant utterance for now
+                example.set(RESPONSE, assistant_utterances[0].get("text"))
```

Comment:
Can we re-use an existing constant like `nlu.constants.TEXT` or define a new one?